internal/lsp/lsproto/_generate/generate.mjs

```diff
@@ -60,19 +60,34 @@ function writeLine(s) {
 
 /**
  * @param {string | undefined} doc
+ * @param {string | undefined} proposed,
+ * @param {string | undefined} since,
+ * @param {string | undefined} sinceTags,
+ * @param {string | undefined} deprecated
  */
-function writeDocumentation(doc) {
+function writeDocumentation(doc, proposed, since, sinceTags, deprecated) {

```

Comment:
This should really just accept a single parameter of a shape that includes all of these props, looking at each of the calls.

Reply:
I swapped this to an inline object type. I looked in `metaModelSchema.mts` to see if there was a base type that each of these objects extended but didn't find one. Let me know if you have a better idea here.